iphone/Classes/TiUINavigationWindowProxy.m

```diff
@@ -220,6 +220,7 @@ - (void)navigationController:(UINavigationController *)navigationController will
     [theWindow windowWillOpen];
     [theWindow windowDidOpen];
   }
+  navController.view.backgroundColor = theWindow.view.backgroundColor;
```

Comment:
It feels like a workaround, since thats not necessary natively. Can you please go through the view tree and check the differences between native views / VC's?